src/components/molecules/CarCard.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import { StyleSheet, View, Text, Image } from "react-native";
+import { LinearGradient } from 'expo-linear-gradient';
+import { Inter_500Medium, Inter_700Bold, Inter_300Light, useFonts } from "@expo-google-fonts/inter";
+
+
+// Define the CarCard component to accept carName, carType, and imageLocation as props
+type CarCardProps = {
+    carName: string;
+    carType: string;
+    imageLocation: any;
+};
+
+export const CarCard = ({ carName, carType, imageLocation }: CarCardProps) => {
+    const [fontsLoaded] = useFonts({
+        Inter_500Medium,
+        Inter_700Bold,
+        Inter_300Light
+    });
```

Comment:
Font loading have been moved to the `App.tsx` file, to decrease the number of times it has to be loaded.

Reply:
Awesome. 
I have removed the loading of the fonts in the component.